index.js

```diff
@@ -44,97 +44,168 @@ formtag.addEventListener('submit', (event) => {
 // projects cards
 const projects = [
   {
-    name: 'Profesional Art Printing Data',
+    name: 'Profesional Art Printing Data 6',
     description:
       "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
-    featuredImages: '',
+    featuredImage: 'images/exc.png',
     technologies: ['HTML', 'BootStrap', 'Ruby'],
     linktoliveversion: '',
     linktosource: '',
   },
   {
-    name: 'Profesional Art Printing Data',
+    name: 'Profesional Art Printing Data 5',
     description:
       "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
-    featuredImages: '',
+    featuredImage: 'images/sada1.png',
     technologies: ['HTML', 'BootStrap', 'Ruby'],
     linktoliveversion: '',
     linktosource: '',
   },
   {
-    name: 'Profesional Art Printing Data',
+    name: 'Profesional Art Printing Data 4',
     description:
```

Comment:
- [ ] The first card is also part of the project section and should be included in this array and loaded to the DOM dynamically.
![afbeelding](https://github.com/Rdzaman20000/Portfolio/assets/39240820/61f388ba-dcd9-4362-b3c8-9c9f29c885ac)